apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/RegexUtils.java

```diff
@@ -0,0 +1,34 @@
+package gov.cms.bfd.server.war.commons;
+
+import com.google.common.collect.ImmutableList;
+
+/** Helper Utils class for Functions shared across. multiple classes. */
+public class RegexUtils {
```

Comment:
Since this has nothing at all to do with regexes can we rename this to `StringUtils` or something?